### DISCUSSION TITLE
Renamed Apple Music songs scrobbling with original metadata

### DISCUSSION THREAD

**Discussion by hujiba** [ciderapp/Cider#1627] [Feature Request]
### Describe the Bug

I used iTunes to change the metadata of a song in my library from apple music. On the library album page in Cider the new metadata is shown, but in the top play bar, and on scrobbles, the original metadata is used.

Renamed album and songs:
![image](https://user-images.githubusercontent.com/34780215/207514012-3e95a02c-b8db-43b0-9d8a-f84beec79d4a.png)

Original metadata showing up:
![image](https://user-images.githubusercontent.com/34780215/207514066-35798177-bc61-45dd-8c54-fa5ca27a3d5d.png)
![image](https://user-images.githubusercontent.com/34780215/207514729-b3c7d29f-e22b-411f-8a50-98d05ab51e30.png)

### Steps to Reproduce

1. Added a song from Apple Music to my library
2. Used iTunes on Windows to change the song title and album
3. Played the song in Cider from my library

### Environment Information

- OS: Windows 11
- Desktop Environment: 
- App Version: 1.6.0
- Commit ID:

### How did you download the software?

Chocolatey

### Code of Conduct

- [X] I agree to follow this Cider's [Code of Conduct](https://github.com/ciderapp/Cider/blob/main/CODE_OF_CONDUCT.md)

### Disclaimer

- [X] I acknowledge the [Support Disclaimer](https://docs.cider.sh/support/disclaimer)

**Comment by coredev-uk** [comment]
More of a feature request than a bug. Converting to discussion.
